qt/python/instrumentview/instrumentview/renderers/test/test_renderers.py

```
@@ -130,9 +136,168 @@ def test_reconstructed_triangles_match_original(self):
+        quad_verts, _ = _extract_quad_from_cylinder_shapeinfo(si)
+        x_min, x_max = quad_verts[:, 0].min(), quad_verts[:, 0].max()
+        self.assertAlmostEqual(x_max - x_min, 2 * radius, places=8)
+
+    def test_wish_cylinder_quad_lies_in_xy_plane(self):
+        """For Y-axis cylinder, quad should lie in the local XY plane (z=0)."""
+        si = self._make_cylinder_si(axis=(0.0, 1.0, 0.0))
+        quad_verts, _ = _extract_quad_from_cylinder_shapeinfo(si)
+        np.testing.assert_allclose(quad_verts[:, 2], 0.0, atol=1e-12)
+
+    def test_wish_cylinder_normal_faces_negative_z(self):
+        """Normal = e1 × e2 should point in the (0, 0, ±1) direction for Y-axis cylinder."""
```

Ruff (0.15.15)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)